iOS/README.md

```diff
@@ -0,0 +1,145 @@
+<p align="center">
```

Comment:
Is it worth having Swift in it's on markdown file and linking to it from the main readme (could even have an internal readme as a contents if you wanted as well).

I don't know much about the word of iOS so don't know how many guidelines would be included in this folder. @simongilmurray may have more input. I've asked him to PR the contents of this work as well.

Reply:
@wordshaker Made requested changes and moved it to Swift folder incase any future guides want to be added to iOS but is not necessarily related to Swift.